pkg/controller/podgc/gc_controller_test.go

```diff
@@ -669,18 +679,18 @@ func verifyDeletedAndPatchedPods(t *testing.T, client *fake.Clientset, wantDelet
 	}
 }
 
-func testDeletingPodsMetrics(t *testing.T, inputDeletingPodsTotal int) {
+func testDeletingPodsMetrics(t *testing.T, total int, reason string) {
 	t.Helper()
 
-	actualDeletingPodsTotal, err := metricstestutil.GetCounterMetricValue(deletingPodsTotal.WithLabelValues())
+	actualDeletingPodsTotal, err := metricstestutil.GetCounterMetricValue(metrics.DeletingPodsTotal.WithLabelValues("", reason))
```

Comment:
What is the namespace for these pods?

Reply:
@xing-yang It's fixed in https://github.com/kubernetes/kubernetes/compare/04ac55e68117dff43adaaf3be47e39578e44dadc..322da7c1aae34496fda5ce5dd96966706d6513b4